docs/source/couchdb_tutorial.rst

```diff
@@ -246,24 +239,14 @@ and refreshed every time new records are added to the state database.
 Add the index to your chaincode folder
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once you finalize an index, it is ready to be packaged with your chaincode for
-deployment by being placed alongside it in the appropriate metadata folder.
-
-If your chaincode installation and instantiation uses the Hyperledger
-Fabric Node SDK, the JSON index files can be located in any folder as long
-as it conforms to this `directory structure <https://hyperledger.github.io/fabric-sdk-node/master/tutorial-metadata-chaincode.html>`__.
-During the chaincode installation using the ``client.installChaincode()`` API,
-include the attribute (``metadataPath``) in the `installation request <https://hyperledger.github.io/fabric-sdk-node/master/global.html#ChaincodeInstallRequest>`__.
-The value of the metadataPath is a string representing the absolute path to the
-directory structure containing the JSON index file(s).
-
-Alternatively, if you are using the
-:doc:`commands/peercommand` command to install and instantiate the chaincode, then the JSON
-index files must be located under the path ``META-INF/statedb/couchdb/indexes``
+Once you finalize an index, you need to package it with your chaincode for
+deployment by being placed alongside it in the appropriate metadata folder. You
```

Comment:
```suggestion
deployment by placing it in the appropriate metadata folder. You
```

This also seems to overlap with the explicit directions that were added below. Seems we would only need one.

Reply:
done.